src/operator/numpy/np_elemwise_broadcast_op.cc

```diff
@@ -182,5 +182,79 @@ MXNET_OPERATOR_REGISTER_NP_BINARY_SCALAR(_npi_rpower_scalar)
 .set_attr<FCompute>("FCompute<cpu>", BinaryScalarOp::Compute<cpu, mshadow_op::rpower>)
 .set_attr<nnvm::FGradient>("FGradient", ElemwiseGradUseOut{"_backward_rpower_scalar"});
 
+MXNET_OPERATOR_REGISTER_BINARY_BROADCAST(_npi_ldexp)
+.describe(R"code(
+    ldexp(x1, x2, out=None)
```

Comment:
No need for docs here as this is now an internal operator.

Reply:
Fixed. Thx.